rca/account_management/views.py

```diff
@@ -164,7 +165,7 @@ def form_valid(self, form):
 class CustomLoginView(LoginView):
     """Custom login view to redirect students to their profile page"""
 
-    template_name = "wagtailadmin/login.html"
+    template_name = "patterns/pages/auth/login.html"
 
     def get_success_url(self):
         """Override the success URL if the user meets the following:
```

Comment:
We should make sure to test this on the dev site, as in logging in via wagtail as a student still works well as I'm not sure this is unit tested

Reply:
Yep - I tested this locally and it was working as intended.